packages/x-date-pickers/src/locales/frFR.ts

```diff
@@ -1,10 +1,12 @@
 import { PickersLocaleText } from './utils/pickersLocaleTextApi';
 import { getPickersLocalization } from './utils/getPickersLocalization';
+import { TimeViewWithMeridiem } from '../internals/models';
```

Comment:
Do you think that it still makes sene in keeping this type internal if we are adding it into locale files? 🤔